src/main/target/SPEEDYBEEF4/target.h

```diff
@@ -0,0 +1,152 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "SBF4"
+#define USBD_PRODUCT_STRING  "SpeedyBeeF4"
+#define USE_TARGET_CONFIG
+
+#define LED0_PIN                PB9
+
+#define BEEPER                  PC13
+#define BEEPER_INVERTED
+
+#define ENABLE_DSHOT_DMAR       true
+
+// *************** Gyro & ACC **********************
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+#define MPU6000_CS_PIN          PB11
+#define MPU6000_SPI_INSTANCE    SPI1
+
+#define USE_EXTI
+#define MPU_INT_EXTI            PC4
+#define USE_MPU_DATA_READY_SIGNAL
+
+#define USE_GYRO
+#define USE_GYRO_SPI_MPU6000
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+
+#define USE_MAG
+#define USE_MAG_HMC5883
```

Comment:
You have added the: `drivers/compass/compass_qmc5883l.c` in the `target.mk` but you don't have added here the 
```
#define USE_MAG_QMC5883
```